acceleration-everest-03-dotnet/Infrastructure.Data/DependencyInjections/InfrastructureDataExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using EntityFrameworkCore.UnitOfWork.Extensions;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Infrastructure.Data.DependencyInjections
+{
+    public static class InfrastructureDataExtensions
```

Comment:
O nome é de acordo com o tipo que você está estendendo 
```suggestion
    public static class IServiceCollectionExtensions
```

Reply:
Certo. Mudo todos então né?